test_project/settings.py

```diff
@@ -186,7 +163,25 @@
         'cities_light': {
             'handlers':['console'],
             'propagate': True,
-            'level':'DEBUG',
+            'level':'WARNING',
         },
     }
 }
+
+if os.environ.get('CI', False):
+    CITIES_LIGHT_TRANSLATION_LANGUAGES=['fr', 'ru']
```

Comment:
Does that mean we will import all languages by default ? That's very, very long ^^